2023/README.md

```diff
@@ -1,20 +1,276 @@
+
+
+### Flexbox
+
+Flexbox is a one-dimensional layout method for arranging items in rows or columns. Items flex (expand) to fill additional space or shrink to fit into smaller spaces. This article explains all the fundamentals.
+
+[https://developer.mozilla.org/docs/Learn/CSS/CSS_layout/Flexbox](https://developer.mozilla.org/docs/Learn/CSS/CSS_layout/Flexbox)
+
+[https://drafts.csswg.org/css-flexbox/](https://drafts.csswg.org/css-flexbox/)
+
+[https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-flexbox](https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-flexbox)
+
+
+### Fonts
+
+Enable testing for font stack capabilities and enable additional expressiveness with vector color fonts. ([Font feature detection and palettes](https://github.com/web-platform-tests/interop/issues/164))
+
+[https://developer.mozilla.org/en-US/docs/Web/CSS/font-palette](https://developer.mozilla.org/en-US/docs/Web/CSS/font-palette)
+
+[https://w3c.github.io/csswg-drafts/css-fonts/#font-palette-prop](https://w3c.github.io/csswg-drafts/css-fonts/#font-palette-prop)
+
+[https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-fonts](https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-fonts)
+
+
+### Forms
+
+The &lt;form> HTML element represents a document section containing interactive controls for submitting information.
+
+[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form)
+
+[https://html.spec.whatwg.org/multipage/forms.html#the-form-element](https://html.spec.whatwg.org/multipage/forms.html#the-form-element)
+
+
+### Grid
+
+CSS Grid Layout excels at dividing a page into major regions or defining the relationship in terms of size, position, and layer, between parts of a control built from HTML primitives.
+
+[https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)
+
+[https://drafts.csswg.org/css-grid/](https://drafts.csswg.org/css-grid/)
+
+[https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-grid](https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-grid)
+
+
+### :has()
+
+With a :host() selector referencing the root of an element with a shadow root, it holds that :host(:has(...) should reference light DOM children of that root. Some browsers acknowledge this truth (Safari), others don't (Chrome), others have yet to ship :has() support (Firefox).
```

Comment:
Oops, this is describing another proposal. Replacing by TODO so this doesn't accidentally make it in:

```suggestion
TODO
```